### ISSUE TITLE
target_group_arns is asking for a list when passing in interpolation syntax

### ISSUE THREAD

**Issue by ulikabbq** [terraform-aws-modules/terraform-aws-autoscaling#4]
I have 
`target_group_arns = ["${data.terraform_remote_state.alb.target_group_arn}"]`

and terraform plan returns 
`module.cb-asg.module.autoscaling_group.aws_autoscaling_group.this: target_group_arns: should be a list`

when I  put it to the output, I get a valid arn 
`output "target_group_arn" {
  value = "${data.terraform_remote_state.alb.target_group_arn}"
}`

then I can copy/paste that arn into  here and it works just fine. 
`target_group_arns = ["arn-from-my-output"]`

**Comment by antonbabenko** [comment]
Use [list() interpolation function](https://www.terraform.io/docs/configuration/interpolation.html#list-items-) like this:
```
target_group_arns = ["${list(data.terraform_remote_state.alb.target_group_arn)}"]
```

**Comment by ulikabbq** [comment]
thanks

**Comment by ulikabbq** [comment]
this is still not working for me

**Comment by antonbabenko** [comment]
@ulikabbq @alexballas has fixed this issue in #5 , new release of this module has been tagged ([v1.0.3](https://registry.terraform.io/modules/terraform-aws-modules/autoscaling/aws/1.0.3)), so you should be able to use it now.
